net-mgmt/zabbix-agent/src/opnsense/service/templates/OPNsense/ZabbixAgent/zabbix_agentd.conf

```diff
@@ -0,0 +1,310 @@
+#
+### Option: UserParameter
+#	User-defined parameter to monitor. There can be several user-defined parameters.
+#	Format: UserParameter=<key>,<shell command>
+#	See 'zabbix_agentd' directory for examples.
+#
+# 	Disabled. A configuration file should be placed on directory: <%= @include %>
+
+####### LOADABLE MODULES #######
+### Option: LoadModulePath
+#	Full path to location of agent modules.
+#	Default depends on compilation options.
+#
+#LoadModulePath=/usr/local/lib/modules
+
+### Option: LoadModule
+#	Module to load at agent startup. Modules are used to extend functionality of the agent.
+#	Format: LoadModule=<module.so>
+#	The modules must be located in directory specified by LoadModulePath.
+#	It is allowed to include multiple LoadModule parameters.
+#
+
+####### TLS-RELATED PARAMETERS #######
+
+### Option: TLSConnect
+#	How the proxy should connect to Zabbix server. Used for an active proxy, ignored on a passive proxy.
+#	Only one value can be specified:
+#		unencrypted - connect without encryption
+#		psk         - connect using TLS and a pre-shared key
+#		cert        - connect using TLS and a certificate
+#
+# Mandatory: yes, if TLS certificate or PSK parameters are defined (even for 'unencrypted' connection)
+# Default:
+# TLSConnect=unencrypted
+
+### Option: TLSAccept
+#	What incoming connections to accept from Zabbix server. Used for a passive proxy, ignored on an active proxy.
+#	Multiple values can be specified, separated by comma:
+#		unencrypted - accept connections without encryption
+#		psk         - accept connections secured with TLS and a pre-shared key
+#		cert        - accept connections secured with TLS and a certificate
+#
+# Mandatory: yes, if TLS certificate or PSK parameters are defined (even for 'unencrypted' connection)
+# Default:
+# TLSAccept=unencrypted
+
+### Option: TLSCAFile
+#	Full pathname of a file containing the top-level CA(s) certificates for
+#	peer certificate verification.
+#
+# Mandatory: no
+# Default:
+# TLSCAFile=
+
+### Option: TLSCRLFile
+#	Full pathname of a file containing revoked certificates.
+#
+# Mandatory: no
+# Default:
+# TLSCRLFile=
+
+### Option: TLSServerCertIssuer
+#      Allowed server certificate issuer.
+#
+# Mandatory: no
+# Default:
+# TLSServerCertIssuer=
+
+### Option: TLSServerCertSubject
+#      Allowed server certificate subject.
+#
+# Mandatory: no
+# Default:
+# TLSServerCertSubject=
+
+### Option: TLSCertFile
+#	Full pathname of a file containing the proxy certificate or certificate chain.
+#
+# Mandatory: no
+# Default:
+# TLSCertFile=
+
+### Option: TLSKeyFile
+#	Full pathname of a file containing the proxy private key.
+#
+# Mandatory: no
+# Default:
+# TLSKeyFile=
+
+### Option: TLSPSKIdentity
+#	Unique, case sensitive string used to identify the pre-shared key.
+#
+# Mandatory: no
+# Default:
+# TLSPSKIdentity=
+
+### Option: TLSPSKFile
+#	Full pathname of a file containing the pre-shared key.
+#
+# Mandatory: no
+# Default:
+# TLSPSKFile=
+
+{% endif %}
```

Comment:
I think it is a good habit to keep comments from the sample configuration to give an idea about a certain value and it's usefulness. [It's](https://github.com/opnsense/core/blob/399b9090a40f9ac728863d21b511b75b76759db3/src/etc/inc/system.inc#L1169-L1177) a [very](https://github.com/theforeman/puppet-foreman/blob/ec8c6c25356429b01cd590158932664271db387e/templates/settings.yaml.erb#L30-L31) [common](https://github.com/theforeman/puppet-puppet/blob/ad08281a610d5667c89c2c2d9d95da38c66d08ca/templates/server/puppetserver/conf.d/puppetserver.conf.erb#L24-L33) [thing](https://github.com/jpds/puppet-strongswan/blob/b0a21f9350d41cda437304ab969fe159aba9c868/templates/charon.conf.erb#L6-L14) to do.